learning_journal/templates/create.jinja2

```diff
@@ -0,0 +1,10 @@
+{% extends "layout.jinja2" %}
+{% block content %}
+    <form method="post">
+        Title: <input value="" name='title'>
+        </br></br>Body:</br>

```

Comment:
using `<br />` tags is generally a sign that you haven't considered updating your css to fix spacing issues.  It would be better to solve layout problems with css than with additional markup.